### DISCUSSION TITLE
Lack of anti-idle function(缺少反空闲功能)

### DISCUSSION THREAD

**Discussion by git1071440198** [Eugeny/tabby#7131] [General]
To prevent disconnection due to idle, you want to increase the anti-idle function, every 600 seconds ( custom time ), automatically enter“ \ n”, “ls” and other custom symbols.（防止由于空闲断开连接，希望增加反空闲功能，每隔600秒(自定义时间)，自动输入\n、ls等自定义符号。）

**Comment by lawweiliang** [comment]
@git1071440198 
English only please. 
https://github.com/Eugeny/tabby/discussions/5533

![image](https://user-images.githubusercontent.com/34936929/228172727-cc58dcc4-bdf4-4085-844a-a25cf784c145.png)
